shared/templates/template_ANSIBLE_mount_option_removable_partitions

```diff
@@ -3,31 +3,11 @@
 # strategy = configure
 # complexity = low
 # disruption = high
-- (xccdf-var {{{ MOUNTPOINT }}})
+- (xccdf-var var_removable_partition)
 
-- name: get back mount information associated to mountpoint
-  command: findmnt --fstab '{{ {{{ MOUNTPOINT }}} }}'
-  register: device_name
-  failed_when: device_name.rc > 1
-  changed_when: False
-
-- name: create mount_info dictionary variable
-  set_fact:
-    mount_info: "{{ mount_info|default({})|combine({item.0: item.1}) }}"
-  with_together:
-    - "{{ device_name.stdout_lines[0].split() | list | lower }}"
-    - "{{ device_name.stdout_lines[1].split() | list }}"
-  when:
-    - device_name.stdout is defined and device_name.stdout_lines is defined
-    - (device_name.stdout | length > 0)
-
-- name: Ensure permission {{{ MOUNTOPTION }}} are set on {{{ MOUNTPOINT }}}
-  mount:
-    path: "{{ {{{ MOUNTPOINT }}} }}"
-    src: "{{ mount_info.source }}"
-    opts: "{{ mount_info.options }},{{{ MOUNTOPTION }}}"
-    state: "mounted"
-    fstype: "{{ mount_info.fstype }}"
-  when:
-    - device_name.stdout is defined
-    - (device_name.stdout | length > 0)
+- name: Ensure permission {{{ MOUNTOPTION }}} are set on var_removable_partition
+  lineinfile:
+    path: /etc/fstab
+    regexp: '^\s*({{ var_removable_partition }})\s+([^\s]*)\s+([^\s]*)\s+([^\s]*)(.*)$'
+    backrefs: yes
+    line: '\1 \2 \3 \4,{{{ MOUNTOPTION }}} \5'
```

Comment:
I'm not sure we should expect `/etc/fstab` to already contain an entry for the removable device in question.
I tend to think the remediation should add it.

Reply:
Probably yes, sounds reasonable

Reply:
But, I have realized another thing now. The rule should pass when `/etc/fstab` doesn't contain the entry. We have a test scenario `no_partitions.pass.sh` and this scenario simply erases all contents of `/etc/fstab`. I understand it was created to cover https://bugzilla.redhat.com/show_bug.cgi?id=1403905. That is an old BZ  where the reporter asks to no fail the scan if the devices are not mounted. I am not sure if the scenario meets his expectation.

Reply:
Is it possible for the device not to be mounted and still have the entry in `/etc/fstab`?

Reply:
I think yes, when the removable media is removed.

Reply:
Concerning e.g. DVDs, if there is no device file, it implies that the computer doesn't have a drive, so there is nothing needed in fstab and the rule should simply pass.
The remediation could add the fstab line if the device file exists, but that's outside of what Ansible can do easily, and we aren't even sure how exactly should the line components look like. So I think that this is OK.

Reply:
The rule passes when /dev/cdrom doesn't exist and this isn't changed by this PR.

However, the rule also passes when /dev/cdrom exists but there is no line with /dev/cdrom in /etc/fstab. This is because of the old bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1403905. See the regression test for this BZ in https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/permissions/partitions/mount_option_noexec_removable_partitions/tests/no_partitions.pass.sh. From the test scenario you can see that we expect the rule to pass when /dev/cdrom exists but it isn't configured in /etc/fstab. I have revisited this old BZ and I think that this interpretation might be incorrect, because the mount option isn't enforced in this situation. Instead, I think that the rule should always require explicit configuration in /etc/fstab. On the other hand, if we change the behavior in this situation, we create a regression for the customer that reported the old BZ.

I don't know if it's possible to add the fstab line when the fstab line doesn't exist, because we don't know to which mountpoint the user wishes to mount the device. Should we use some predefined directory eg. /media/cdrom?

Reply:
Thanks for your answer - the request in that BZ is really strange. However, as you point out, that rule doesn't have a remediation for cases when the entry in `/etc/fstab` is missing, which means no all-green state after the remediation. Creating a remediation that would fix `fstab` and that wouldn't introduce an antipattern is out of scope of this PR, as nobody complains about that aspect of the check.